### ISSUE TITLE
Tesla.Env.__struct__/0 is undefined

### ISSUE THREAD

**Issue by yevhene** [tallarium/reverse_proxy_plug#139]
Can't compile 1.5.0 anymore. 1.4.0 is ok.

```
==> reverse_proxy_plug
Compiling 9 files (.ex)

== Compilation error in file lib/reverse_proxy_plug/http_client/adapters/tesla.ex ==
** (CompileError) lib/reverse_proxy_plug/http_client/adapters/tesla.ex:33: Tesla.Env.__struct__/0 is undefined, cannot expand struct Tesla.Env. Make sure the struct name is correct. If the struct name exists and is correct but it still cannot be found, you likely have cyclic module usage in your code
    (stdlib 3.13.2) lists.erl:1358: :lists.mapfoldl/3
    (stdlib 3.13.2) lists.erl:1359: :lists.mapfoldl/3
could not compile dependency :reverse_proxy_plug, "mix compile" failed. You can recompile this dependency with "mix deps.compile reverse_proxy_plug", update it with "mix deps.update reverse_proxy_plug" or clean it with "mix deps.clean reverse_proxy_plug"
```

**Comment by axelson** [comment]
Note: this is fixed by either #140 or #141

**Comment by yevhene** [comment]
@axelson Glad to hear, please release a version with a fix

**Comment by axelson** [comment]
I'm not the maintainer, but perhaps a patch release will be coming soon :crossed_fingers:

**Comment by Fedjmike** [comment]
@yevhene I've released v2.0.0, which includes the fix.
